
    <!DOCTYPE html>
    <html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Pyrosequencing</title>
        <link rel="stylesheet" href="wiki.css">
    </head>
    <body>
        <div class="container">
            <section class="intro">
                <h2>Pyrosequencing</h2>
                <p>Pyrosequencing is a method of DNA sequencing (determining the order of nucleotides in DNA) based on the "sequencing by synthesis" principle, in which the sequencing is performed by detecting the nucleotide incorporated by a DNA polymerase. Pyrosequencing relies on light detection based on a chain reaction when pyrophosphate is released. Hence, the name pyrosequencing.</p>
<p>The principle of pyrosequencing was first described in 1993 by, Bertil Pettersson, Mathias Uhlen and Pål Nyren by combining the solid phase sequencing method using streptavidin coated magnetic beads with recombinant DNA polymerase lacking 3´to 5´exonuclease activity (proof-reading) and luminescence detection using the firefly luciferase enzyme.  A mixture of three enzymes (DNA polymerase, ATP sulfurylase and firefly luciferase) and a nucleotide (dNTP) are added to single stranded DNA to be sequenced and the incorporation of nucleotide is followed by measuring the light emitted. The intensity of the light determines if 0, 1 or more nucleotides have been incorporated, thus showing how many complementary nucleotides are present on the template strand. The nucleotide mixture is removed before the next nucleotide mixture is added. This process is repeated with each of the four nucleotides until the DNA sequence of the single stranded template is determined.</p>
<p>A second solution-based method for pyrosequencing was described in 1998 by Mostafa Ronaghi, Mathias Uhlen and Pål Nyren. In this alternative method, an additional enzyme apyrase is introduced to remove nucleotides that are not incorporated by the DNA polymerase. This enabled the enzyme mixture including the DNA polymerase, the luciferase and the apyrase to be added at the start and kept throughout the procedure, thus providing a simple set-up suitable for automation. An automated instrument based on this principle was introduced to the market the following year by the company Pyrosequencing.</p>
<p>A third microfluidic variant of the pyrosequencing method was described in 2005 by Jonathan Rothberg and co-workers at the company 454 Life Sciences. This alternative approach for pyrosequencing was based on the original principle of attaching the DNA to be sequenced to a solid support and they showed that sequencing could be performed in a highly parallel manner using a microfabricated microarray. This allowed for high-throughput DNA sequencing and an automated instrument was introduced to the market. This became the first next generation sequencing instrument starting a new era in genomics research, with rapidly falling prices for DNA sequencing allowing whole genome sequencing at affordable prices.</p>
<p>Procedure</p>
<p>"Sequencing by synthesis" involves taking a single strand of the DNA to be sequenced and then synthesizing its complementary strand enzymatically. The pyrosequencing method is based on detecting the activity of DNA polymerase (a DNA synthesizing enzyme) with another chemoluminescent enzyme.  Essentially, the method allows sequencing a single strand of DNA by synthesizing the complementary strand along it, one base pair at a time, and detecting which base was actually added at each step.  The template DNA is immobile, and solutions of A, C, G, and T nucleotides are sequentially added and removed from the reaction.  Light is produced only when the nucleotide solution complements the first unpaired base of the template.  The sequence of solutions which produce chemiluminescent signals allows the determination of the sequence of the template.</p>
<p>For the solution-based version of pyrosequencing, the single-strand DNA (ssDNA) template is hybridized to a sequencing primer and incubated with the enzymes DNA polymerase, ATP sulfurylase, luciferase and apyrase, and with the substrates adenosine 5´ phosphosulfate (APS) and luciferin.</p>
<p>The addition of one of the four deoxynucleotide triphosphates (dNTPs) (dATPαS, which is not a substrate for a luciferase, is added instead of dATP to avoid noise) initiates the second step. DNA polymerase incorporates the correct, complementary dNTPs onto the template. This incorporation releases pyrophosphate (PPi).</p>
<p>ATP sulfurylase converts PPi to ATP in the presence of adenosine 5´ phosphosulfate. This ATP acts as a substrate for the luciferase-mediated conversion of luciferin to oxyluciferin that generates visible light in amounts that are proportional to the amount. The light produced in the luciferase-catalyzed reaction is detected by a camera and analyzed in a program.</p>
<p>Unincorporated nucleotides and ATP are degraded by the apyrase, and the reaction can restart with another nucleotide.</p>
<p>The process can be represented by the following equations:</p>
<p>PPi + APS → ATP + Sulfate (catalyzed by ATP-sulfurylase);</p>
<p>ATP + luciferin + O2 → AMP + PPi + oxyluciferin + CO2 + hv (catalyzed by luciferase);</p>
<p>where:</p>
<p>PPi is pyrophosphate</p>
<p>APS is adenosine 5-phosphosulfate;</p>
<p>ATP is adenosine triphosphate;</p>
<p>O2 is oxygen molecule;</p>
<p>AMP is adenosine monophosphate;</p>
<p>CO2 is carbon dioxide;</p>
<p>hv is light.</p>
<p>Limitations</p>
<p>Currently, a limitation of the method is that the lengths of individual reads of DNA sequence are in the neighborhood of 300-500 nucleotides, shorter than the 800-1000 obtainable with chain termination methods (e.g. Sanger sequencing).  This can make the process of genome assembly more difficult, particularly for sequences containing a large amount of repetitive DNA. Lack of proof-reading activity limits accuracy of this method.</p>
<p>Commercialization</p>
<p>The company Pyrosequencing AB in Uppsala, Sweden was founded with venture capital provided by HealthCap in order to commercialize machinery and reagents for sequencing short stretches of DNA using the pyrosequencing technique. Pyrosequencing AB was listed on the Stockholm Stock Exchange in 1999. It was renamed to Biotage in 2003. The pyrosequencing business line was acquired by Qiagen in 2008.  Pyrosequencing technology was further licensed to 454 Life Sciences. 454 developed an array-based pyrosequencing technology which emerged as a platform for large-scale DNA sequencing, including genome sequencing and metagenomics.</p>
<p>Roche announced the discontinuation of the 454 sequencing platform in 2013.</p>
<p>References</p>
<p>Further reading</p>
<p>Metzker M. (2005). "Emerging Technologies in DNA Sequencing". Genome Research. 15 (12): 1767–76. doi:10.1101/gr.3770505. PMID 16339375.</p>

            </section>
        </div>
    </body>
    </html>
    