
    <!DOCTYPE html>
    <html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Transdifferentiation</title>
        <link rel="stylesheet" href="wiki.css">
    </head>
    <body>
        <div class="container">
            <section class="intro">
                <h2>Transdifferentiation</h2>
                <p>Transdifferentiation, also known as lineage reprogramming, is the process in which one mature somatic cell is transformed into another mature somatic cell without undergoing an intermediate pluripotent state or progenitor cell type. It is a type of metaplasia, which includes all cell fate switches, including the interconversion of stem cells. Current uses of transdifferentiation include disease modeling and drug discovery and in the future may include gene therapy and regenerative medicine. The term 'transdifferentiation' was originally coined by Selman and Kafatos in 1974 to describe a change in cell properties as cuticle producing cells became salt-secreting cells in silk moths undergoing metamorphosis.</p>
<p>Discovery</p>
<p>Davis et al. 1987 reported the first instance (sight) of transdifferentiation where a cell changed from one adult cell type to another. Forcing mouse embryonic fibroblasts to express MyoD was found to be sufficient to turn those cells into myoblasts.</p>
<p>Natural examples</p>
<p>The only known instances where adult cells change directly from one lineage to another occurs in the species Turritopsis dohrnii (also known as the immortal jellyfish) and Turritopsis nutricula.</p>
<p>In newts, when the eye lens is removed, pigmented epithelial cells de-differentiate and then redifferentiate into the lens cells. Vincenzo Colucci described this phenomenon in 1891 and Gustav Wolff described the same thing in 1894; the priority issue is examined in Holland (2021). </p>
<p>In humans and mice, it has been demonstrated that alpha cells in the pancreas can spontaneously switch fate and transdifferentiate into beta cells. This has been demonstrated for both healthy and diabetic human and mouse pancreatic islets. While it was previously believed that oesophageal cells were developed from the transdifferentiation of smooth muscle cells, that has been shown to be false.</p>
<p>Induced and therapeutic examples</p>
<p>The first example of functional transdifferentiation has been provided by Ferber et al. by inducing a shift in the developmental fate of cells in the liver and converting them into 'pancreatic beta-cell-like' cells. The cells induced a wide, functional and long-lasting transdifferentiation process that reduced the effects of hyperglycemia in diabetic mice. Moreover, the trans-differentiated beta-like cells were found to be resistant to the autoimmune attack that characterizes type 1 diabetes.</p>
<p>The second step was to undergo transdifferentiation in human specimens. By transducing liver cells with a single gene, Sapir et al. were able to induce human liver cells to transdifferentiate into human beta cells.</p>
<p>This approach has been demonstrated in mice, rat, xenopus and human tissues.</p>
<p>Schematic model of the hepatocyte-to-beta cell transdifferentiation process. Hepatocytes are obtained by liver biopsy from diabetic patient, cultured and expanded ex vivo, transduced with a PDX1 virus, transdifferentiated into functional insulin-producing beta cells, and transplanted back into the patient.</p>
<p>Granulosa and theca cells in the ovaries of adult female mice can transdifferentiate to Sertoli and Leydig cells via induced knockout of the FOXL2 gene. Similarly, Sertoli cells in the testes of adult male mice can transdifferentiate to granulosa cells via induced knockout of the DMRT1 gene.</p>
<p>Methods</p>
<p>Lineage-instructive approach</p>
<p>In this approach, transcription factors from progenitor cells of the target cell type are transfected into a somatic cell to induce transdifferentiation. There exists two different means of determining which transcription factors to use: by starting with a large pool and narrowing down factors one by one or by starting with one or two and adding more. One theory to explain the exact specifics is that ectopic Transcriptional factors direct the cell to an earlier progenitor state and then redirects it towards a new cell type. Rearrangement of the chromatin structure via DNA methylation or histone modification may play a role as well. Here is a list of in vitro examples and in vivo examples. In vivo methods of transfecting specific mouse cells utilize the same kinds of vectors as in vitro experiments, except that the vector is injected into a specific organ. Zhou et al. (2008) injected Ngn3, Pdx1 and Mafa into the dorsal splenic lobe (pancreas) of mice to reprogram pancreatic exocrine cells into β-cells in order to ameliorate hyperglycaemia.</p>
<p>Initial epigenetic activation phase approach</p>
<p>Somatic cells are first transfected with pluripotent reprogramming factors temporarily (Oct4, Sox2, Nanog, etc.) before being transfected with the desired inhibitory or activating factors. Here is a list of examples in vitro.</p>
<p>Pharmacological agents</p>
<p>The DNA methylation inhibitor, 5-azacytidine is also known to promote phenotypic transdifferentiation of cardiac cells to skeletal myoblasts.</p>
<p>In prostate cancer, treatment with androgen receptor targeted therapies induces neuroendocrine transdifferentiation in a subset of patients. No standard of care exists for these patients, and those diagnosed with treatment induced neuroendocrine carcinoma are typically treated palliatively.</p>
<p>Mechanism of action</p>
<p>The transcription factors serve as a short term trigger to an irreversible process. The transdifferentiation liver cells observed 8 months after one single injection of pdx1.</p>
<p>The ectopic transcription factors turn off the host repertoire of gene expression in each of the cells. However, the alternate desired repertoire is being turned on only in a subpopulation of predisposed cells. Despite the massive dedifferentiation – lineage tracing approach indeed demonstrates that transdifferentiation originates in adult cells.</p>
<p>Mogrify algorithm</p>
<p>Determining the unique set of cellular factors that is needed to be manipulated for each cell conversion is a long and costly process that involved much trial and error. As a result, this first step of identifying the key set of cellular factors for cell conversion is the major obstacle researchers face in the field of cell reprogramming. An international team of researchers have developed an algorithm, called Mogrify(1), that can predict the optimal set of cellular factors required to convert one human cell type to another. When tested, Mogrify was able to accurately predict the set of cellular factors required for previously published cell conversions correctly. To further validate Mogrify's predictive ability, the team conducted two novel cell conversions in the laboratory using human cells, and these were successful in both attempts solely using the predictions of Mogrify. Mogrify has been made available online for other researchers and scientists.</p>
<p>Issues</p>
<p>Evaluation</p>
<p>When examining transdifferentiated cells, it is important to look for markers of the target cell type and the absence of donor cell markers which can be accomplished using green fluorescent protein or immunodetection. It is also important to examine the cell function, epigenome, transcriptome, and proteome profiles. Cells can also be evaluated based upon their ability to integrate into the corresponding tissue in vivo and functionally replace its natural counterpart. In one study, transdifferentiating tail-tip fibroblasts into hepatocyte-like cells using transcription factors Gata4, Hnf1α and Foxa3, and inactivation of p19(Arf) restored hepatocyte-like liver functions in only half of the mice using survival as a means of evaluation.</p>
<p>Transition from mouse to human cells</p>
<p>Generally transdifferentiation that occurs in mouse cells does not translate in effectiveness or speediness in human cells. Pang et al. found that while transcription factors Ascl1, Brn2 and Myt1l turned mouse cells into mature neurons, the same set of factors only turned human cells into immature neurons. However, the addition of NeuroD1 was able to increase efficiency and help cells reach maturity.</p>
<p>Order of transcription factor expression</p>
<p>The order of expression of transcription factors can direct the fate of the cell. Iwasaki et al. (2006) showed that in hematopoietic lineages, the expression timing of Gata-2 and (C/EBPalpha) can change whether or not a lymphoid-committed progenitors can differentiate into granulocyte/monocyte progenitor, eosinophil, basophil or bipotent basophil/mast cell progenitor lineages.</p>
<p>Immunogenicity</p>
<p>It has been found for induced pluripotent stem cells that when injected into mice, the immune system of the synergeic mouse rejected the teratomas forming. Part of this may be because the immune system recognized epigenetic markers of specific sequences of the injected cells. However, when embryonic stem cells were injected, the immune response was much lower. Whether or not this will occur within transdifferentiated cells remains to be researched.</p>
<p>Method of transfection</p>
<p>In order to accomplish transfection, one may use integrating viral vectors such as lentiviruses or retroviruses, non-integrating vectors such as Sendai viruses or adenoviruses, microRNAs and a variety of other methods including using proteins and plasmids; one example is the non-viral delivery of transcription factor-encoding plasmids with a polymeric carrier to elicit neuronal transdifferentiation of fibroblasts. When foreign molecules enter cells, one must take into account the possible drawbacks and potential to cause tumorous growth. Integrating viral vectors have the chance to cause mutations when inserted into the genome. One method of going around this is to excise the viral vector once reprogramming has occurred, an example being Cre-Lox recombination Non-integrating vectors have other issues concerning efficiency of reprogramming and also the removal of the vector. Other methods are relatively new fields and much remains to be discovered.</p>
<p>Differences with pluripotent reprogramming</p>
<p>Almost all factors that reprogram cells into pluripotency have been discovered and can turn a wide variety of cells back into induced pluripotent stem cells (iPSCs). However, many of the reprogramming factors that can change a cell's lineage have not been discovered and these factors apply only for that specific lineage.</p>
<p>The final products of transdifferentiated cells are capable of being used for clinical studies, but iPSCs must be differentiated.</p>
<p>It may become possible in the future to use transdifferentiation in vivo, whereas pluripotent reprogramming may cause teratomas in vivo.</p>
<p>Transdifferentiated cells will require less epigenetic marks to be reset, whereas pluripotent reprogramming requires nearly all to be removed, which may become an issue during redifferentiation.</p>
<p>Transdifferentiation is geared towards moving between similar lineages, whereas pluripotent reprogramming has unlimited potential.</p>
<p>Pluripotent cells are capable of self-renewal and often go through many cell passages, which increases the chance of accumulating mutations. Cell culture may also favor cells that are adapted for surviving under those conditions, as opposed to inside an organism. Transdifferentiation requires fewer cell passages and would reduce the chance of mutations.</p>
<p>Transdifferentiation can also be much more efficient than pluripotency reprogramming due to the extra step involved in the latter process.</p>
<p>Both pluripotent and transdifferentiated cells use adult cells, thus starting cells are very accessible, whereas human embryonic stem cells require that one navigate legal loopholes and delve into the morality of stem cell research debate.</p>
<p>See also</p>
<p>Epigenetics</p>
<p>Induced pluripotent stem cell</p>
<p>Induced stem cells</p>
<p>Reprogramming</p>
<p>== References ==</p>

            </section>
        </div>
    </body>
    </html>
    